### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by paulwarren-wk**
This PR addresses accessibility issues listed on this report:

https://www.webaccessibility.com/results/?url=https%3A%2F%2Fwww.workiva.com%2Fsites%2Fworkiva%2Ffiles%2F2020-01%2Fixbrl_viewer-45fa445867ef4917ab19c7fad4c1f507-results_4%2Fixbrl-viewer.htm

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js [hunk 1]
```diff
@@ -80,14 +80,27 @@ iXBRLViewer.prototype.pluginPromise = function (methodName, ...args) {
...
+    else {
+        docTitle = "Inline Viewer";
+    }
+    if ($('html').attr("lang") === undefined) {
+        $('html').attr("lang", "en-US");
+    }
+
     $('head').children().not("script").not("style#ixv-style").not("link#ixv-favicon").appendTo($(iframe).contents().find('head'));
+
+    $('<title>').text(docTitle != "" ? "Inline Viewer - " + docTitle : "Inline Viewer").appendTo($('head'));
```
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js:103]: I think just `docTitle` (the `!= ""` is already done above?).

**Review by brettkail-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Review by sagesmith-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10

@Workiva/release-management-p

**Comment by derekgengenbacher-wf** [comment]
QA +10

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**